src/main/java/net/dzikoysk/funnyguilds/util/elo/EloConstant.java

```diff
@@ -0,0 +1,22 @@
+package net.dzikoysk.funnyguilds.util.elo;
+
+public class EloConstant {
+
+    private int minRank;
```

Comment:
Zmienne mozna finalne dac!

Reply:
Co ty tu chcesz finalnego dawać?

Reply:
Zmienne, toć to one się zmieniać nie będą.

Reply:
A, bo tam setter'a w końcu nie ma, to fakt